sdks/go/sdk.go

```diff
@@ -57,7 +58,7 @@ func NewSDK() (*SDK, error) {
 	// Block for at least 30 seconds.
 	ctx, cancel := context.WithTimeout(s.ctx, 30*time.Second)
 	defer cancel()
-	conn, err := grpc.DialContext(ctx, addr, grpc.WithBlock(), grpc.WithInsecure())
+	conn, err := grpc.DialContext(ctx, addr, grpc.WithBlock(), grpc.WithTransportCredentials(insecure.NewCredentials()))
```

Comment:
This change is added because `make lint` failed with the following error.
```
sdks/go/sdk.go:60:61: SA1019: grpc.WithInsecure is deprecated: use WithTransportCredentials and insecure.NewCredentials() instead. Will be supported throughout 1.x. (staticcheck)
        conn, err := grpc.DialContext(ctx, addr, grpc.WithBlock(), grpc.WithInsecure())
```